cmd/aries-js-worker/src/aries.js

```diff
@@ -176,90 +176,255 @@ const Aries = function(opts) {
             return () => {quit = true}
         },
 
+        /**
+         * DIDExchange methods - Refer to [OpenAPI spec](docs/rest/openapi_spec.md#generate-openapi-spec) for
+         * input params and output return json values.
+         */
         didexchange: {
             pkgname: "didexchange",
-            createInvitation: async function (text) {
+
+            /**
+             * Creates a DID Exchange Invitation.
+             *
+             * @param req - json document
+             * @returns {Promise<String>}
+             */
+            createInvitation: async function (req) {
                 return invoke(aw, pending,  this.pkgname, "CreateInvitation", text, "timeout while creating invitation")
             },
+
+            /**
+             * Receives a DID Exchange invitation.
+             *
+             * @param invitation - json document
+             * @returns {Promise<String>}
+             */
             receiveInvitation: async function (invitation) {
                 return invoke(aw, pending,  this.pkgname, "ReceiveInvitation", invitation, "timeout while receiving invitation")
             },
-            acceptInvitation: async function (connectionID, publicDID="") {
+
+            /**
+             * Accepts a DID Exchange invitation.
+             *
+             * @param req - json document
+             * @returns {Promise<String>}
+             */
+            acceptInvitation: async function (req) {
                 return new Promise((resolve, reject) => {
-                    invoke(aw, pending,  this.pkgname, "AcceptInvitation", {id: connectionID, public: publicDID}, "timeout while accepting invitation").then(
+                    invoke(aw, pending,  this.pkgname, "AcceptInvitation", req, "timeout while accepting invitation").then(
```

Comment:
Reverted API param changes done in https://github.com/hyperledger/aries-framework-go/pull/1357/files#diff-a43de5ddb6dcf66800367a9232d43f90R187 as index.html doesnt support multiple params. Need to revisit this after v0.1.2 version.

Reply:
Issue number?

Reply:
https://github.com/hyperledger/aries-framework-go/issues/1366